nuget/nunit.runners.xamarin.nuspec

```diff
@@ -43,6 +43,9 @@ Supported Xamarin platforms:
         <file src="src\runner\nunit.runner.wp81\bin\Release\nunit.runner.wp81.dll" target="lib\wpa81\nunit.runner.wp81.dll" />
         <file src="src\runner\nunit.runner.wp81\bin\Release\nunit.runner.wp81.xml" target="lib\wpa81\nunit.runner.wp81.xml" />
         
+        <file src="nuget\uap10.0\MainPage.xaml.txt.pp" target="content\uap10.0\MainPage.xaml.txt.pp" />
+        <file src="nuget\uap10.0\MainPage.xaml.cs.txt.pp" target="content\uap10.0\MainPage.xaml.cs.txt.pp" />
```

Comment:
I removed these because they weren't getting pulled into UWP projects. Did you test the package you created with a test project? Do these get pulled in?